adafruit_character_lcd/character_lcd.py

```diff
@@ -693,15 +708,15 @@ def color(self):
         return self._color
 
     @color.setter
-    def color(self, color):
+    def color(self, color: Union[List[int], int]) -> None:
```

Comment:
`circuitpython_typing` also has a `Protocol` for colors I think applies here and would fit better for `color`.

Reply:
I just had a look at the [circuitpython_typing docs](https://docs.circuitpython.org/projects/adafruit-circuitpython-typing/en/latest/api.html#circuitpython-typing) and I couldn't find a `color`.

Reply:
I think this is what we want:  https://github.com/adafruit/Adafruit_CircuitPython_Typing/blob/dfe89ba609bf98a5138d76744669e8f513a0d30b/circuitpython_typing/led.py#L18

Reply:
Ignore the above comment

Reply:
So perhaps we go with `Union[int, List[float, float, float]]` ? Or `Union[int, Union[List[float, float, float], Tuple[float, float, float]]]` ? Since a Tuple would work equally well.